server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/api/posttask/PostProjectAnalysisTasksExecutor.java

```diff
@@ -207,9 +220,10 @@ private BranchImpl createBranch() {
     @Nullable
     private final Analysis analysis;
 
-    private ProjectAnalysisImpl(CeTask ceTask, Project project,
+    private ProjectAnalysisImpl(@Nullable Organization organization, CeTask ceTask, Project project,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 8 parameters, which is greater than 7 authorized. ([squid:S00107](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fsl%2FSONAR-10223%2Fadd_organization_to_PostProjectAnalysisTask&issues=AWCYjonG_Q-0dz4JAW3H&open=AWCYjonG_Q-0dz4JAW3H)

[ISSUE_KEY]: AWCYjonG_Q-0dz4JAW3H